datacommons/places.py

```diff
@@ -92,3 +92,59 @@ def get_places_in(dcids, place_type):
   if isinstance(dcids, pd.Series):
     return pd.Series([result[dcid] for dcid in dcids])
   return result
+
+
+def get_pop_obs(dcid):
+  """ Returns :obj:`Place`'s population and observations.
+
+  Args:
+    dcids (:obj:`str`): Dcid of the place.
+
+  Returns:
+    An dictionary of population and observation related to this place.
+
+  Raises:
+    ValueError: If the payload returned by the Data Commons REST API is
+      malformed.
+
+  Examples:
+    We would like to get population and observation of Mountain View
```

Comment:
:obj:\`StatisticalPopulation\`'s and :obj:\`Observations\`'s of \`Mountain View <https://browser.datacommons.org/kg?dcid=geoId/0649670>\`_.